package.json

```diff
@@ -8,11 +8,12 @@
     "preversion": "npm run clean && npm run check",
     "postversion": "git push && git push --tags && npm run clean && npm run docs:publish",
     "version": "npm run build",
-    "clean": "rimraf build",
+    "clean": "rimraf build dist",
     "lint": "eslint src/**",
     "test": "mocha --compilers js:babel/register --recursive src/**/__tests__/*.js",
     "check": "npm run lint && npm run test:all",
     "build": "babel src --out-dir build",
+    "build:amd": "webpack",
```

Comment:
why do we need all of webpack in order to produce a umd build? we should be able to do that with browserify pretty trivially